src/components/application_manager/test/application_manager_impl_test.cc

```diff
@@ -795,13 +813,15 @@ TEST_F(ApplicationManagerImplTest,
   const auto hmi_level_switching_app = mobile_apis::HMILevel::HMI_FULL;
   EXPECT_CALL(*switching_app_ptr, hmi_level())
       .WillRepeatedly(Return(hmi_level_switching_app));
+  EXPECT_CALL(*switching_app_ptr, app_id()).WillRepeatedly(Return(0u));
```

Comment:
@ValeriiMalkov change to ON_CALLS

Reply:
@LuxoftAKutsan Fixed in commit cb0fa9a5bd7649dc7322d420554dc10fdaf4adff